pom.xml

```diff
@@ -36,6 +36,20 @@
     <prerequisites>
         <maven>2.2.0</maven>
     </prerequisites>
+
```

Comment:
Of course, you're right. It's only about the change in SwaggerOperationController.java.